apps/studio/src/lib/db/clients/cockroach.ts

```diff
@@ -0,0 +1,143 @@
+import globals from "@/common/globals";
+import pg, { PoolConfig } from "pg";
+import { IDbConnectionDatabase, IDbConnectionServer, IDbConnectionServerConfig } from "../types";
+import { SupportedFeatures, TableIndex, TablePartition, TableProperties, TableTrigger } from "../models";
+import { PostgresClient } from "./postgresql";
+import _ from 'lodash';
+
+
+export class CockroachClient extends PostgresClient {
+
+  constructor(server: any, database: IDbConnectionDatabase) {
```

Comment:
I think we can skip writing `constructor` if it's the same as the parent, no?

Reply:
Yeah I originally had some extra configuration in there and just forgot to remove it after I refactored. 🤦